### PR TITLE
Added admin security requirement.

### PR DESCRIPTION
**Pull request by Malind19**

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/remotedesktop/get-rdsessioncollection.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Get-RDSessionCollection [[-CollectionName] <String>] [-ConnectionBroker <String>
 The **Get-RDSessionCollection** cmdlet gets session collections in a Remote Desktop deployment.
 A session collection consists of one or more Remote Desktop Session Host (RD Session Host) servers.
 Users can connect to RD Session Host servers in a session collection to run programs, save files, and use resources on those servers.
+User needs to be a member of the machine Administrator group to run this command.
```
Comment (JohanFreelancer9) [inline docset/windows/remotedesktop/get-rdsessioncollection.md]: **The user** needs to be a member of the machine Administrator group to run this command. 

Please do change as indicated in **bold**.

**Comment by Malind19** [comment]
@mypil applied copy review changes here.

**Comment by Malind19** [comment]
@e0i Please check

**Pull request merged**